--- slim.original	2014-05-19 05:24:30.132381987 +0800
+++ slim	2014-05-19 05:24:42.092441298 +0800
@@ -26,6 +26,16 @@ SSD_STOP_ARGS="--pidfile $PIDFILE --name
 
 case $1 in
   start)
+    if [ -f /root/.firstRun ];then
+        cd /usr/share/slim/themes/cubian-default
+        /bin/ln -sf panel-text.png panel.png
+    else
+        panel=$(readlink /usr/share/slim/themes/cubian-default/panel.png)
+        if [ "$panel" = "panel-text.png" ];then
+            cd /usr/share/slim/themes/cubian-default
+            /bin/ln -sf panel-normal.png panel.png
+        fi
+    fi
     if [ "$HEED_DEFAULT_DISPLAY_MANAGER" = "true" ] &&
        [ -e $DEFAULT_DISPLAY_MANAGER_FILE ] &&
        [ "$(cat $DEFAULT_DISPLAY_MANAGER_FILE)" != "$DAEMON" ]; then
@@ -35,6 +45,9 @@ case $1 in
       start-stop-daemon --start --quiet $SSD_START_ARGS || echo -n " already running"
       echo "."
     fi
+    if [ -f /root/.firstRun ];then
+        rm -f /root/.firstRun
+    fi
   ;;
 
   stop)
