--- XScreenSaver.original	2013-11-14 00:12:52.776121516 +0000
+++ XScreenSaver	2013-11-14 00:10:46.726187248 +0000
@@ -29,15 +29,15 @@
 ! /* (xrdb prevention kludge: whole file)
 
 *mode:			random
-*timeout:		0:10:00
+*timeout:		0:15:00
 *cycle:			0:10:00
 *lockTimeout:		0:00:00
-*passwdTimeout:		0:00:30
+*passwdTimeout:		0:00:10
 *dpmsEnabled:		False
 *dpmsQuickoffEnabled:	False
-*dpmsStandby:		2:00:00
-*dpmsSuspend:		2:00:00
-*dpmsOff:		4:00:00
+*dpmsStandby:		0:00:00
+*dpmsSuspend:		0:00:00
+*dpmsOff:		0:00:00
 *grabDesktopImages:	True
 *grabVideoFrames:	False
 *chooseRandomImages:	False
