--- udevil.conf.original	2014-02-22 21:35:36.030308548 +0800
+++ udevil.conf	2014-02-22 21:41:01.859924249 +0800
@@ -101,7 +101,7 @@ allowed_groups = *
 # be permitted to unmount its associated loop device even though internal.
 # INCLUDING /MNT HERE IS NOT RECOMMENDED.  ALL ALLOWED MEDIA DIRECTORIES
 # SHOULD BE OWNED AND WRITABLE ONLY BY ROOT.
-allowed_media_dirs = /media, /run/media/$USER
+allowed_media_dirs = /media, /run/media/$USER, /home/$USER
 
 
 # allowed_devices is the first criteria for what block devices users may mount
@@ -133,7 +133,7 @@ allowed_devices = /dev/*
 # Some removable esata drives look like internal drives to udevil.  To avoid
 # this problem, they can be treated as removable with this setting.
 # WARNING:  SETTING A SYSTEM DEVICE HERE CAN CAUSE SERIOUS SECURITY PROBLEMS.
-# allowed_internal_devices =
+allowed_internal_devices = /dev/sd*,/dev/nand*
 
 
 # allowed_internal_uuids and allowed_internal_uuids_FSTYPE work similarly to
@@ -217,10 +217,10 @@ default_options_file      = nosuid, noex
 # mount iso9660 with 'ro' to prevent mount read-only warning
 default_options_iso9660   = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID, ro, utf8
 default_options_udf       = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID
-default_options_vfat      = nosuid, noexec, nodev, noatime, fmask=0022, dmask=0022, uid=$UID, gid=$GID, utf8
-default_options_msdos     = nosuid, noexec, nodev, noatime, fmask=0022, dmask=0022, uid=$UID, gid=$GID
-default_options_umsdos    = nosuid, noexec, nodev, noatime, fmask=0022, dmask=0022, uid=$UID, gid=$GID
-default_options_ntfs      = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID, utf8
+default_options_vfat      = nosuid, noexec, nodev, noatime, fmask=0000, dmask=0000, uid=$UID, gid=$GID, utf8
+default_options_msdos     = nosuid, noexec, nodev, noatime, fmask=0000, dmask=0000, uid=$UID, gid=$GID
+default_options_umsdos    = nosuid, noexec, nodev, noatime, fmask=0000, dmask=0000, uid=$UID, gid=$GID
+default_options_ntfs      = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID, nls=utf8
 default_options_cifs      = nosuid, noexec, nodev, uid=$UID, gid=$GID
 default_options_smbfs     = nosuid, noexec, nodev, uid=$UID, gid=$GID
 default_options_sshfs     = nosuid, noexec, nodev, noatime, uid=$UID, gid=$GID, nonempty, allow_other
@@ -241,7 +241,7 @@ default_options_ramfs     = nosuid, noex
 # and GID.
 # If you want to forbid remounts, remove 'remount' from here.
 # WARNING:  OPTIONS HERE CAN CAUSE SERIOUS SECURITY PROBLEMS - CHOOSE CAREFULLY
-allowed_options           = nosuid, noexec, nodev, noatime, fmask=0022, dmask=0022, uid=$UID, gid=$GID, ro, rw, sync, flush, iocharset=*, utf8, remount
+allowed_options           = nosuid, noexec, nodev, noatime, fmask=*, dmask=*, uid=$UID, gid=$GID, ro, rw, sync, flush, iocharset=*, nls=*, utf8, remount
 allowed_options_nfs       = nosuid, noexec, nodev, noatime, ro, rw, sync, remount, port=*, rsize=*, wsize=*, hard, proto=*, timeo=*, retrans=*
 allowed_options_cifs      = nosuid, noexec, nodev, ro, rw, remount, port=*, user=*, username=*, pass=*, password=*, guest, domain=*, uid=$UID, gid=$GID, credentials=*
 allowed_options_smbfs     = nosuid, noexec, nodev, ro, rw, remount, port=*, user=*, username=*, pass=*, password=*, guest, domain=*, uid=$UID, gid=$GID, credentials=*
