--- ifplugd.action.origin	2013-08-25 21:00:08.520962184 +0800
+++ ifplugd.action	2013-08-25 21:01:54.405487235 +0800
@@ -22,6 +22,10 @@
 case "$2" in
 up)
 	run-parts --arg="$1" --arg="$2" /etc/ifplugd/action.d/
+    ifdelay=$(date --date='5 seconds' +"%m%d%H%M.%S")
+    at -f /usr/local/bin/cubian-networklight-wrapper -t $ifdelay
+    dhcpdelay=$(date --date='20 seconds' +"%m%d%H%M.%S")
+    at -f /usr/sbin/ntpdate-debian -t $dhcpdelay
 	;;
 down)
 	run-parts --reverse --arg="$1" --arg="$2" /etc/ifplugd/action.d/
