--- motd.original	2014-01-20 03:20:42.517463845 +0800
+++ motd	2014-01-20 03:23:47.550381293 +0800
@@ -1,4 +1,13 @@
 
+      ______  __    __  .______    __       ___      .__   __. 
+     /      ||  |  |  | |   _  \  |  |     /   \     |  \ |  | 
+    |  ,----'|  |  |  | |  |_)  | |  |    /  ^  \    |   \|  | 
+    |  |     |  |  |  | |   _  <  |  |   /  /_\  \   |  . `  | 
+    |  `----.|  `--'  | |  |_)  | |  |  /  _____  \  |  |\   | 
+     \______| \______/  |______/  |__| /__/     \__\ |__| \__| 
+                                                              
+                        http://cubian.org
+
 The programs included with the Debian GNU/Linux system are free software;
 the exact distribution terms for each program are described in the
 individual files in /usr/share/doc/*/copyright.
