--- slim.conf.original	2014-05-30 00:19:00.821773237 +0800
+++ slim.conf	2014-05-30 00:25:24.555676074 +0800
@@ -34,7 +34,11 @@ authfile           /var/run/slim.auth
 # to adjust the command according to your preferred shell,
 # i.e. for freebsd use:
 # login_cmd           exec /bin/sh - ~/.xinitrc %session
-login_cmd           exec /bin/bash -login /etc/X11/Xsession %session
+login_cmd             exec /bin/bash -login /usr/lib/cubian/cubian-xinitrc %session %screen_width %screen_height %theme
+
+# This command is executed after a succesful login.
+# it runs in root
+login_success_cmd     exec /bin/bash /usr/lib/cubian/cubian-xinitrc-success %screen_width %screen_height
 
 # Commands executed when starting and exiting a session.
 # They can be used for registering a X11 session with
@@ -52,7 +56,8 @@ login_cmd           exec /bin/bash -logi
 # The current chosen session name is replaced in the login_cmd
 # above, so your login command can handle different sessions.
 # see the xinitrc.sample file shipped with slim sources
-sessions            default,startxfce4,openbox,ion3,icewm,wmaker,blackbox,awesome
+#sessions            default,startxfce4,openbox,ion3,icewm,wmaker,blackbox,awesome
+sessions            lxde
 
 # Executed when pressing F11 (requires imagemagick)
 screenshot_cmd      scrot /root/slim.png
@@ -82,10 +87,11 @@ reboot_msg         The system is rebooti
 
 # current theme, use comma separated list to specify a set to 
 # randomly choose from
-current_theme       debian-joy
+current_theme       theme-current
 
 # Lock file
 lockfile            /var/run/slim.lock
+coverlockfile       /var/lock/slim_disable_cover.lock
 
 # Log file
 logfile             /var/log/slim.log
