--- slim.conf.original	2014-07-06 19:33:58.779319983 +0800
+++ slim.conf	2014-07-07 00:08:43.153416299 +0800
@@ -23,7 +23,7 @@ authfile           /var/run/slim.auth
 
 # Hide the mouse cursor (note: does not work with some WMs).
 # Valid values: true|false
-# hidecursor          false
+hidecursor          true
 
 # This command is executed after a succesful login.
 # you can place the %session and %theme variables
@@ -34,7 +34,15 @@ authfile           /var/run/slim.auth
 # to adjust the command according to your preferred shell,
 # i.e. for freebsd use:
 # login_cmd           exec /bin/sh - ~/.xinitrc %session
-login_cmd           exec /bin/bash -login /etc/X11/Xsession %session
+login_cmd             exec /bin/bash -login /usr/lib/cubian/cubian-xinitrc %session %screen_width %screen_height %theme
+
+# This command is executed after a succesful login.
+# it runs in root
+login_success_cmd     exec /bin/bash /usr/lib/cubian/cubian-xinitrc-success %screen_width %screen_height 
+
+# This command is executed when execute when using "slim -c".
+# it runs in root
+prelogin_cmd          /usr/bin/xterm -maximized -C -fg white -bg black +sb -T "Cubian Config" -e /bin/bash -c "/usr/sbin/cubian-config"
 
 # Commands executed when starting and exiting a session.
 # They can be used for registering a X11 session with
@@ -52,7 +60,8 @@ login_cmd           exec /bin/bash -logi
 # The current chosen session name is replaced in the login_cmd
 # above, so your login command can handle different sessions.
 # see the xinitrc.sample file shipped with slim sources
-sessions            default,startxfce4,openbox,ion3,icewm,wmaker,blackbox,awesome
+#sessions            default,startxfce4,openbox,ion3,icewm,wmaker,blackbox,awesome
+sessions            lxde
 
 # Executed when pressing F11 (requires imagemagick)
 screenshot_cmd      scrot /root/slim.png
@@ -82,10 +91,11 @@ reboot_msg         The system is rebooti
 
 # current theme, use comma separated list to specify a set to 
 # randomly choose from
-current_theme       debian-joy
+current_theme       theme-current
 
 # Lock file
 lockfile            /var/run/slim.lock
+coverlockfile       /var/lock/slim_disable_cover.lock
 
 # Log file
 logfile             /var/log/slim.log
